DemoApp/DemoAppCoordinator.swift

```diff
@@ -103,9 +107,11 @@ final class DemoAppCoordinator: NSObject, UNUserNotificationCenterDelegate {
         Components.default.messageContentView = CustomMessageContentView.self
         Components.default.messageListDateSeparatorEnabled = true
         Components.default.messageListDateOverlayEnabled = true
-        Components.default._messageListDiffingEnabled = true
+        Components.default._messageListDiffingEnabled = isStreamInternalConfiguration
```

Comment:
We will soon move this to the default as well, but either way, I don't think there is any problem if customers use this in our demo app 🤔